src/atmos_param/qe_moist_convection/qe_moist_convection.F90

```diff
@@ -422,6 +423,7 @@ subroutine CAPE_calculation(k_surface, p_full, p_half, Tin, rin, kLZB, &
     Tp     = Tin
     rp     = rin
     saturated = .false.
+    kLCL = 0
```

Comment:
Was this just something that was never explicitly initialised before?

Reply:
yup; kLCL was added to qe_moist_convection in the spookie p/r, but is also needed for the column model